ts-tests/scripts/export_env.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+export etherscan="RF71W4Z2RDA7XQD6EN19NGB66C2QD9UPHB"
```

Comment:
hard code api tokens?

Reply:
Yes, it's a good point. We had this since the litentry node test. I would write down an issue to encrypt the config file like this one.